src/anvil/mod.rs

```diff
@@ -0,0 +1,139 @@
+use std::borrow::Cow;
+
+use testcontainers::{
+    core::{ContainerPort, WaitFor},
+    Image,
+};
+
+const NAME: &str = "ghcr.io/foundry-rs/foundry";
+const TAG: &str = "latest";
```

Comment:
We prefer to avoid  `latest` tag as it can easily lead to a module being broken (e.g. an incompatible version is released).

Users always can override it with [ImageExt::with_tag](https://docs.rs/testcontainers/0.23.1/testcontainers/core/trait.ImageExt.html#tymethod.with_tag) in their code.

Reply:
Good point! The [Foundry package tags](https://github.com/foundry-rs/foundry/releases) are a bit unfriendly in this regard so I've gone with `stable` but with the sha as well. Hope that does the job.